GOVERNANCE.md

```diff
@@ -7,12 +7,21 @@ This document defines project governance for the Strimzi project.
 ### Maintainers
 
 The maintainers are responsible for the overall development of the whole Strimzi project and all its components.
-The maintainers are identified in the [`MAINTAINERS`](MAINTAINERS) file. 
+Emeritus maintainers are former maintainers who are no longer actively involved in the project.
+In the rest of this document the term ‘maintainer’ will be assumed to mean ‘active maintainer’, unless qualified as ‘emeritus maintainer’.
+Both maintainers and emeritus maintainers are identified in the [`MAINTAINERS`](MAINTAINERS) file. 
 
 #### Changes in Maintainership
 
 New maintainers are proposed by an existing maintainer and are elected by a ⅔ majority maintainers vote.
-Maintainers can be removed by a ⅔ majority maintainers vote.
+Maintainers can be removed by a ⅔ majority maintainers vote. 
+Maintainers removed by a vote are no longer considered maintainers or emeritus maintainers.
+Maintainers become emeritus maintainers either actively, by opening a PR against the MAINTAINERS file marking themselves as Emeritus, or implicitly after 6 months without any contribution to the project.
+Updates to the MAINTAINERS to mark implicitly Emeritus maintainers happen asynchronously (i.e. the MAINTAINERS file is not normative about Emeritus status).
+An Emeritus maintainer reverts to being an active maintainer by contributing to the project again.
```

Comment:
```suggestion
An Emeritus maintainer reverts to being an active maintainer through consistent and sustained contributions to the project again.
The MAINTAINERS file is then adjusted accordingly.
```

Reply:
Should this be six months of sustained contributions, like six months of inactivity becoming an emeritus maintainer?